cli/main.py

```diff
@@ -52,6 +58,29 @@ def get_access_management_rows(
     return []
 
 
+def get_data_masking_rows(
+    manifest: Manifest,
+    data_masking_config: DataMaskingConfig,
+    project_name: str,
+    database_name: str = None,
+) -> List[DataMaskingRow]:
+    sql_engine = manifest.metadata.adapter_type
```

Comment:
Right now this code is same as in `get_access_management_rows` but different function is being called. I think we can refactor those two methods to accept `manifest_nodes` as a parameter instead of `manifest`